### PR TITLE
Translated to Turkish

### PR DESCRIPTION
**Pull request by GurcanH**

### CODE REVIEW

File: content/docs/addons.md [hunk 1]
```diff
@@ -4,46 +4,46 @@ title: Add-Ons
...
-You can install the add-ons individually from npm (e.g. `npm install react-addons-create-fragment`) and import them:
+Eklentileri NPM aracılığyla ayrı ayrı yükleyebilir (örnek: `npm install react-addons-create-fragment`) ve onları içeri aktarabilirsiniz:
 
 ```javascript
 import createFragment from 'react-addons-create-fragment'; // ES6
 var createFragment = require('react-addons-create-fragment'); // ES5 with npm
 ```
 
-When using React 15 or earlier from a CDN, you can use `react-with-addons.js` instead of `react.js`:
+When using React 15 or earlier from a CDN, you can use React 15 ya da daha önceki sürümleri bir CDN'den kullanırken `react.js` yerine  `react-with-addons.js` kullanabilirsiniz:
```
Comment (alioguzhan) [inline content/docs/addons.md:43]: I think you forgot to delete the English part here.

File: content/docs/addons.md [hunk 2]
```diff
@@ -4,46 +4,46 @@ title: Add-Ons
 permalink: docs/addons.html
 ---
 
-> Note:
+> Not:
 >
-> `React.addons` entry point is deprecated as of React v15.5. The add-ons have moved to separate modules, and some of them have been deprecated.
+> `React.addons` giriş noktası React v15.5'ten itibaren kullanımdan kaldırılmıştır.. Bu eklentiler ayrı modüllere taşındı ve bazıları kullanımdan kaldırıldı.
```
Comment (alioguzhan) [inline content/docs/addons.md:9]: not 2 periods :

> `.` instead of `..`

File: content/docs/addons.md [hunk 3]
```diff
@@ -4,46 +4,46 @@ title: Add-Ons
...
 
-The React add-ons are a collection of useful utility modules for building React apps. **These should be considered experimental** and tend to change more often than the core.
+React eklentileri, React uygulamaları oluşturmak için faydalı ve işe yarar modüller koleksiyonudur. **Bunlar deney amaçlı düşünülmeli** ve core'dan daha sık değişime eğilimi olduğu göz önüne alınmalıdır.
```
Comment (alioguzhan) [inline content/docs/addons.md:11]: we can say `React'in çekirdeğinden` instead of `core`. What do you think? @GurcanH

**Review by alioguzhan — CHANGES_REQUESTED** [review]
Thanks for the translation. Please see my comments above.

File: content/docs/addons.md (hunk 1 above)
Comment (GurcanH) [reply content/docs/addons.md:43]: Yes, @alioguzhan, good call! I skipped the delete English part. I'll fix it now.

File: content/docs/addons.md (hunk 2 above)
Comment (GurcanH) [reply content/docs/addons.md:9]: Weird! I can't see in this my code! @alioguzhan

File: content/docs/addons.md (hunk 3 above)
Comment (GurcanH) [reply content/docs/addons.md:11]: makes sense @alioguzhan

**Comment by alioguzhan** [comment]
Thanks for the fixes and translation! Merging it ... @GurcanH

**Comment by GurcanH** [comment]
> Thanks for the fixes and translation! Merging it ... @GurcanH

Thank you @alioguzhan

**Pull request merged**